packages/reactivity/src/reactiveEffect.ts

```diff
@@ -40,15 +40,17 @@ export function track(target: object, type: TrackOpTypes, key: unknown) {
     if (!dep) {
       depsMap.set(key, (dep = createDep(() => depsMap!.delete(key))))
     }
-    if (__DEV__) {
```

Comment:
I think this change is unnecessary, since the explicit if else would produce a smaller production build.

Reply:
This does not affect prod build, rollup has tree shake for it.

Reply:
Oh great, that's good to know.